mappings/net/minecraft/client/recipe/book/ResultCollection.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS cwg net/minecraft/client/recipe/book/ResultCollection
```

Comment:
RecipeResultCollection?

Reply:
Sure, why not. At one point it was RecipeBookCraftableResultRecipeCollection